shared/src/main/scala/async/Async.scala

```diff
@@ -51,19 +50,32 @@ object Async:
   /** Execute asynchronous computation `body` on currently running thread. The thread will suspend when the computation
     * waits.
     */
-  def blocking[T](body: Async ?=> T)(using support: AsyncSupport, scheduler: support.Scheduler): T =
+  def blocking[T](body: Async.Spawnable ?=> T)(using support: AsyncSupport, scheduler: support.Scheduler): T =
     group(body)(using Blocking(CompletionGroup.Unlinked))
 
   /** The currently executing Async context */
   inline def current(using async: Async): Async = async
 
-  def group[T](body: Async ?=> T)(using async: Async): T =
+  /** [[Async.Spawnable]] is a special subtype of [[Async]], also capable of spawning runnable [[Future]]s.
+    *
+    * Most functions should not take [[Spawnable]] as a parameter, unless the function explicitly wants to spawn
+    * "dangling" runnable [[Future]]s. Instead, functions should take [[Async]] and spawn scoped futures within
+    * [[Async.group]].
+    */
+  opaque type Spawnable <: Async = Async
```

Comment:
In capability speak, I would prefer the name `Spawn`, because a method takes the capability to spawn, not something that can be spawned.